### PR TITLE
modifyed readme.md file 2.0

### PR DESCRIPTION
**Pull request by ankitrajrj**
@Deveesh-Shetty i have updated Css is fun Discription . added a new section "Purpose for Creating Css Is Fun"  along with this  i have created a new line in bottom to go on top css is fun line. Create some changes in  Installation &  Contributing  guid  . i hope this make sence

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,13 +40,15 @@ To use the CSS animations and effects in your project, simply download or clone
 
 <br> 
 
-<p align="center"><a href="#snippets">^^ Back to Top ^^</a></p>
+<p align="center"><a href="#snippets">^^ Back to Snippets Top ^^</a></p>
 
 ## Contributing
 
-We welcome contributions to the CSS-Is-Fun project!🚀 For detailed instructions on how to contribute, please refer to the CONTRIBUTING.md file.
+We welcome contributions .Welcome to CSS is Fun on GitHub! We're excited for your contributions, big or small. Join our inclusive community and work on various CSS tasks to improve design and functionality. Let's collaborate and make something great together!🚀 For detailed instructions on how to contribute, please refer to the CONTRIBUTING.md file.
```
Comment (Deveesh-Shetty) [inline README.md:47]: Here the fullstop is with Welcome like this` .Welcome` 

and `welcome` seems repetitive 

Make it like this

```diff
+ We encourage contributions. Welcome
- We welcome contributions .Welcome
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
-# CSS Is Fun
-### Yes! You heard it right CSS is awesome, and this repository contains CSS UI Snippets
-CSS-Is-Fun is a collection of creative and fun CSS animations and effects to enhance your web projects.
+# CSS Is Fun 
+
+### Yes! You heard it right CSS is awesome, That's great to hear! CSS is a powerful tool for designing and styling web pages this repository also contains UI snippets are pre-built CSS code blocks that can be easily integrated into a website to add style and functionality.
```
Comment (Deveesh-Shetty) [inline README.md:3]: Here

... this repository also contains UI snippets `which` are pre-built CSS ...

Ig this sounds better


Just commit these two changes which i mentioned.... rest all looks good ✨

**Review by Deveesh-Shetty — COMMENTED** [review]
I have mentioned some changes in the code fix them, and rest all looks good....

**Comment by ankitrajrj** [comment]
I have updated it again .. please check ✅

File: README.md (hunk 2 above)
Comment (ankitrajrj) [reply README.md]: @Deveesh-Shetty i have fixed it . check now

**Comment by Deveesh-Shetty** [comment]
One small change is required ig u missed it... Rest all is good....

File: README.md [hunk 3]
```diff
@@ -32,13 +40,15 @@ To use the CSS animations and effects in your project, simply download or clone
 
 <br> 
 
-<p align="center"><a href="#snippets">^^ Back to Top ^^</a></p>
+<p align="center"><a href="#snippets">^^ Back to Snippets Top ^^</a></p>
 
 ## Contributing
 
-We welcome contributions to the CSS-Is-Fun project!🚀 For detailed instructions on how to contribute, please refer to the CONTRIBUTING.md file.
+We welcome contributions . Welcome to CSS is Fun on GitHub! We're excited for your contributions, big or small. Join our inclusive community and work on various CSS tasks to improve design and functionality. Let's collaborate and make something great together!🚀 For detailed instructions on how to contribute, please refer to the CONTRIBUTING.md file.
```
Comment (Deveesh-Shetty) [inline README.md]: Here i meant to put space after fullstop

Make it like this -

We encourage your contributions. Welcome to

**Review by Deveesh-Shetty — APPROVED** [review]
LGTM 🚀

Congratulations on your first contribution. ✨✨

**Pull request merged**